src/en/charms-deploying.md

```diff
@@ -237,7 +259,7 @@ juju retry-provisioning 3 27 5
 
 ## Considerations
 
-Charms are running without any separation, so its entirely possible for Charms
+Charms are running without any separation, so its entirely possible for charms
 to stomp all over each others configuration files and try to bind to the same
```

Comment:
I agree but I had to stop reviewing stuff I didn't set out to review. If you propose a good alternative I'll surely included it.